### DISCUSSION TITLE
weekly/2024-03-10

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#188] [Announcements]
# weekly/2024-03-10

① Rolldown 开源 - Rollup 的锈化版 ② 前端调试工具超全汇总 ③ OhMyLive2D - Live2D Web 看板组件 ④ 100万级安装量的开源作品，一年有多少打赏收入？ ⑤ Web-Check - 开源的网站分析工具 ⑥ Safari 提供原生 HTML Switch 控件 ⑦ Simple Icons - 流行标志的 SVG 图标 ⑧ CSS 中玩转 Infinity

https://sugarat.top/weekly/2024-03-10.html

**Comment by ATQQ** [comment]
test
